bkrdoc/analysis/linter/l_single_rules.py

```diff
@@ -41,6 +45,17 @@ def constraint_check(self, constraint_met, error_msg):
                 self.add_error(msg=error_msg)
                 return
 
+    def check_deprecated_commands(self):
+
+        for line in self._list:
+            if line.argname in self.deprecated_commands:
```

Comment:
Other way is to write it:
 if line.argname not in self.deprecated_commands:
   continue

.... continue with code.

With this attitude you can avoid bigger indentations